packages/osl-operator-bundle-image/generated/bundle/manifests/sonataflow.org_sonataflowclusterplatforms.yaml

```diff
@@ -0,0 +1,138 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.16.4
+  creationTimestamp: null
+  name: sonataflowclusterplatforms.sonataflow.org
+spec:
+  group: sonataflow.org
+  names:
+    kind: SonataFlowClusterPlatform
```

Comment:
Silly question, but not blocker.
How are the SonataFlow CRDs synchronized in cases where there are changes.
Imagine a new field is added in the SonataFlow CRD in the sonataflow-operator package?

Reply:
Kubernetes updates the API and overrides it based on the version. So, we are at `v1alpha08`. Any instances will also be updated to the new CR, and the reconciliation will expect to have these new changes. It's the responsibility of the operator developer to handle them.

Once we upgrade to, let's say, `v1beta08` or `v1beta1`, the operator/bundle can support both versions simultaneously, but again, our reconciliation logic must handle it nicely.